Pod/Classes/Cards/Expiry.swift

```diff
@@ -144,18 +144,19 @@ extension Expiry: CustomStringConvertible {
  
  - returns: The date with month and year and time set to one minute before the following month.
  */
-private func toDate(month: UInt, year: UInt) -> Date? {
+private func toDate(_ month: UInt, year: UInt) -> Date? {
```

Comment:
Maybe we should rename this to `dateWith(month: UInt, year: UInt)`? To comply with this rule:

* [Prefer method and function names that make use sites form grammatical English phrases.](https://swift.org/documentation/api-design-guidelines/#strive-for-fluent-usage)